hw6_task_2.py

```diff
@@ -0,0 +1,45 @@
+import math
+
+if __name__ == "__main__":
+    def find_square_data(side: int):
```

Comment:
all definitions should be outside if __name__ == __main__ 
in this block only calls